src/components/VisaProgressBar.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  VisaProgress,
+  FormConfig,
+  SectionName,
+  getOverallPromptIndex,
+} from '@lib/domain/form'
+import { Qualifications } from '@lib/domain/qualifications'
+import { isPromptCompleted } from '@lib/domain/prompts'
+export function VisaProgressBar({
+  config,
+  qualifications,
+}: {
+  config: FormConfig
+  qualifications: Qualifications
+}) {
+  const totalPrompts = Object.values(config.sections).reduce(
```

Comment:
Here we'll want to base it on the sections specified in `config.order`, because only the sections inside `order` are rendered.